lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -248,6 +252,14 @@ def commit(action, post, authorization = nil)
         )
       end
 
+      def json_error(raw_response)
+        {
+          'error_message' => 'Invalid response received from the Global Collect API.  Please contact SOMEONE if you continue to receive this message.' +
+            "  (The raw response returned by the API was #{raw_response.inspect})",
```

Comment:
I'm trying to figure out if it's possible that the contents here would reach the customer un-scrubbed... 🤔

Reply:
Just to clarify, you are saying the contents _should_ be scrubbed and it might not be? Or the other way around?

Reply:
@curiousepic is the complete response gets scrubbed I don't see why having `raw_response.inspect` would cause an issue.

Reply:
@curiousepic @bpollack @nfarve New test to confirm scrubbing in the `message` revealed it may not have been scrubbed, though I had to get creative with the error response I tested with. Updated the `scrub` method to cover bases, in any case.

Reply:
Also, I modeled this solution after other gateways that also use `raw_response.inspect` in error messages, and those do not test errors for scrubbing either. May be worth checking for those other gateways.